### ISSUE TITLE
Detect ".::" before a code block

### ISSUE THREAD

**Issue by JelleZijlstra** [sphinx-contrib/sphinx-lint#63]
<img width="314" alt="Screen Shot 2023-04-27 at 7 16 07 AM" src="https://user-images.githubusercontent.com/906600/234873354-4fd36253-e1c9-4c39-8fbb-66bf65ed6dad.png">

https://docs.python.org/3.12/library/dis.html#opcode-POP_TOP (going to send a PR soon to fix this one)

The markup here is

```

   Removes the top-of-stack item.::

      STACK.pop()

```

Perhaps we should detect the sequence `.::` before a code block and warn.
